### ISSUE TITLE
Can we have White for the "Primary Color"?

### ISSUE THREAD

**Issue by harshitpdoshi** [ccrama/Slide#2818]
Slide version: 5.8.7
Android version: 8.1.0

Can we have a White (as in the Light option for the Base theme) color for the "Primary Color" setting?

**Comment by PiwwowPants** [comment]
I do not believe any primary colors currently make the text on the top bar go from white to black. So that'd be an addition required too, for anyone else theorizing this change.  

I think this would be more than a trivial undertaking unless that logic is already orphaned somewhere or is easily extendable from other existing logic.

More of a reason for anyone else wanting this on why it isn't immediately available. Not that I'm saying it shouldn't be a choice for themes.
